openface_ros/scripts/face_recognition_node

```diff
@@ -206,4 +207,9 @@ if __name__ == '__main__':
         rospy.loginfo('loading face database from %s', db)
         openface_ros.restore_trained_faces(db)
 
+    updater = diagnostic_updater.Updater()
+    updater.setHardwareID("none")
+    updater.add(diagnostic_updater.Heartbeat())
+    rospy.Timer(rospy.Duration(1), lambda event: updater.update())
```

Comment:
Please use the force_update method, the update method is already throttled on 1hz